circle.yml

```diff
@@ -15,9 +15,10 @@ general:
   build_dir: st2-packages
   artifacts:
     - ~/packages
-  notify:
-    webhooks:
-      - url: https://webhooks.stackstorm.net:8531/webhooks/build/events
+
+notify:
+  webhooks:
+    - url: https://webhooks.stackstorm.net:8531/webhooks/build/events
```

Comment:
```yaml
- url: ${ST2_HOOK_URL}
```
and then you can set `ST2_HOOK_URL` value in Circle UI (Environment variables section of Settings) to avoid disclosing that URL in public.